src/Configuration/Dao.php

```diff
@@ -46,19 +46,32 @@ class Dao extends Model\Dao\PimcoreLocationAwareConfigDao
      */
     private const LEGACY_FILE = 'datahub-configurations.php';
 
+    /**
+     * @deprecated Will be removed in Pimcore 11
```

Comment:
It will be removed as soon as we drop 10.6 support, right? So it is not 100% correct. Could you please check other deprecations too? 

```suggestion
     * @deprecated Will be removed as soon as Pimcore 10.6 isn´t supported anymore.
```